src/main/resources/create-references-context.xml

```diff
@@ -25,11 +25,11 @@
 	public MySQLAdaptor(String host,String database,String username,String password,int port)
 	-->
 	<bean id="dbAdapter" name="dbAdapter" class="org.gk.persistence.MySQLAdaptor" scope="singleton">
-		<constructor-arg name="host" index="0" value="${database.host}"/>
-		<constructor-arg name="database" index="1" value="${database.name}"/>
-		<constructor-arg name="username" index="2" value="${database.user}"/>
-		<constructor-arg name="password" index="3" value="${database.password}"/>
-		<constructor-arg name="port" index="4" value="${database.port}"/>
+		<constructor-arg name="host" index="0" value="${release.database.host}"/>
```

Comment:
A note: The other properties files and Spring context files in this project that _currently_ do not have a `release.` prefix may cause confusion to future developers. I recommend changing other properties/context files to match this new naming convention.

Reply:
Yes, as discussed I will be making sure that the changed configs are seen in the other config file(s). Thanks!

Reply:
This has now been completed. Now all 3 context files (application, create-references, and file-download) have matching configurations